pyouroboros/notifiers.py

```diff
@@ -60,4 +60,4 @@ def send(self, container_tuples=None, socket=None, kind='update', next_run=None,
         body = '\r\n'.join(body_fields)
 
         if self.apprise.servers:
-            self.apprise.notify(title=title, body=body)
+            self.apprise.notify(title=title, body=body, body_format=apprise.common.NotifyFormat.TEXT)
```

Comment:
consider dropping reference to `.common`:
i.e.: `self.apprise.notify(title=title, body=body, body_format=apprise.NotifyFormat.TEXT)`

It's just a bit cleaner; but what you have will work fine. - up to you.